res/pbrlut.hpp

```diff
@@ -0,0 +1,367 @@
+#ifndef PBRLUT_HPP
+#define PBRLUT_HPP
+extern int GGX_E_size[3];
+extern float GGX_E[32/*alpha*/][32/*cos_theta*/];
```

Comment:
Is that first dimension mapped to GGX alpha or surface 'roughness'? comment here says it's 'alpha'